sdk/storage/Azure.Storage.Common/tests/Shared/TestExceptionPolicy.cs

```diff
@@ -8,56 +8,57 @@ namespace Azure.Storage.Common.Test
 {
     public class TestExceptionPolicy : HttpPipelinePolicy
     {
-        public List<string> Hosts { get; private set; }
+        public List<string> HostsSetInRequests { get; private set; }
```

Comment:
Is this effectively the same as `TrackedRequestMethods`?   Should it have a similar name?

Reply:
The TrackedRequestMethods is the list of request methods for which we should track the hosts and count the number of failures. The HostsSetInRequests are the actual hosts that are used in the requests that fall under the tracked request methods.